packages/plugin-ext/src/main/node/paths/plugin-paths-service.ts

```diff
@@ -17,13 +17,17 @@
 import { injectable, inject } from 'inversify';
 import { FileSystem, FileStat } from '@theia/filesystem/lib/common';
 import * as path from 'path';
+import {  readdir, remove} from 'fs-extra';
```

Comment:
I'm not sure why it wasn't picked up by our formatting rules, but there seems to be an extra leading `whitespace`.
If I format the file the content gets updated (which means anyone in the future will have this line as a change).

Reply:
btw I would suggest evaluating using prettier for formatting and a linter only for semantic errors. You can setup a auto formatter on a pre-commit hook using "husky"
and never worry about formatting issues in review ever again.

- https://prettier.io/

Reply:
Yes, I've seen husky before, I think the fact we haven't set up something like it yet is due to priority but it certainly looks like a nice tool to have.

Reply:
Here is an example package.json with:

- prettier
- husky pre-commit hook for prettier
- Conventional Angular Commit messages
- Automatic changeLog generation (per package).

https://github.com/SAP/xml-tools/blob/master/package.json

BTW: if you ever integrate prettier you will need to use

- https://github.com/prettier/tslint-config-prettier

to disable all syntax formatting rules in TS-Lint to avoid conflicts.

Perhaps this would be of interest to you...